src/core/client/framework/lib/bootstrap/createManaged.tsx

```diff
@@ -120,28 +196,38 @@ export default async function createContext({
 
   const localeBundles = await generateBundles(locales, localesData);
 
+  const localStorage =
+    (pym && inIframe && createPymStorage(pym, "localStorage")) ||
```

Comment:
Maybe possible to clean this up a bit? It's a bit hard to follow.

Reply:
Sure